### PR TITLE
Ac 15 customer can see updated inventory once payment is done

### PR DESCRIPTION
**Pull request by Shivanithoutireddy**
Please review my code

### CODE REVIEW

File: ac_posystem/src/pages/Cart.js [hunk 1]
```diff
@@ -1,89 +1,155 @@
...
 
-  return (
-    <div className="CartContainer">
-      <div className="Header">
-        <h3 className="Heading">Shopping Cart</h3>
-        <h5 className="Action">Remove all</h5>
-      </div>
+  var storageArr = [];
+  var products;
+  //console.log("JSON.parse(localStorage.getItem(",JSON.parse(localStorage.getItem("cartList1")))
```
Comment (wazmaaali) [inline ac_posystem/src/pages/Cart.js]: remove unnecessary lines of code.

File: ac_posystem/src/pages/Cart.js [hunk 2]
```diff
@@ -1,89 +1,155 @@
...
+
+  return (
+    <div className="cart-parent">
+      <div style={{ display: "flex", justifyContent: "center", textAlign: "center" }}>
+       
+        {/* <h1>All Products</h1>
+        <h1></h1> */}
       </div>
+      <div className="cart-container">
+        {/* <div className="left">
```
Comment (wazmaaali) [inline ac_posystem/src/pages/Cart.js]: remove unnecessary code

File: ac_posystem/src/pages/Cart.js [hunk 3]
```diff
@@ -5,86 +5,109 @@ import "../assets/css/style.css";
...
-  //const products = JSON.parse(localStorage.getItem("cartList1"));
-  localStorage.getItem("cartList1") == null ? products = [] : products = JSON.parse(localStorage.getItem("cartList1"))
-
   const [cartItems, setCartItems] = useState(products);
-  const itemsPrice = cartItems.reduce((a, c) => a + c.c_id * c.sc_price, 0);
   const shippingPrice = 10;
+
+  // Retrieving data from session storage, if data is null, then assign empty array else assign to variable (used terinary operator)
+  sessionStorage.getItem("items") == null ? (products = []) : (products = JSON.parse(sessionStorage.getItem("items")));
+  const itemsPrice = cartItems.reduce((a, c) => a + c.c_id * c.sc_price, 0);
```
Comment (AkhilPJ1) [inline ac_posystem/src/pages/Cart.js:20]: Please use understandable naming for c_id and sc_price.

File: ac_posystem/src/pages/Cart.js [hunk 4]
```diff
@@ -5,86 +5,109 @@ import "../assets/css/style.css";
...
   };
 
   return (
     <div className="cart-parent">
-      <div style={{ display: "flex", justifyContent: "center", textAlign: "center" }}>
-       
-        {/* <h1>All Products</h1>
-        <h1></h1> */}
+      <div
+        style={{
```
Comment (AkhilPJ1) [inline ac_posystem/src/pages/Cart.js:103]: Use the style in CSS

File: ac_posystem/src/pages/Cart.js [hunk 5]
```diff
@@ -108,7 +131,7 @@ function Cart() {
                     -
                   </button>{" "}
                   <div style={{ paddingLeft: 20, paddingRight: 20 }}>
-                    {item.c_id} x ${item.sc_price}
+                    {item.count} x ${item.sc_price}
```
Comment (AkhilPJ1) [inline ac_posystem/src/pages/Cart.js:134]: Please add comments on the functionality

File: ac_posystem/src/pages/Cart.js [hunk 6]
```diff
@@ -5,86 +5,109 @@ import "../assets/css/style.css";
...
+      if (check) {
+
+        // Find particular index and updating the count for that particular product id
+        var index = products.findIndex((x) => x.sc_id === product.sc_id);
+
+        // Decrement product count from array for particular product
+        products[index].count = products[index].count - 1;
+
+        // Update new array values to session storage
+        sessionStorage.setItem("items", JSON.stringify(products));
```
Comment (AkhilPJ1) [inline ac_posystem/src/pages/Cart.js:88]: Instead of section storage, local storage is more secure.

**Review by AkhilPJ1 — COMMENTED** [review]
Comments are given in the code.

**Pull request merged**